### PR TITLE
Add Dame Stephanie "Steve" Shirley to names

### PR DESCRIPTION
**Pull request by justincormack**
Born in Germany, she had to flee on the kindertransport to England in
1939. In the 1950s she worked at the Post Office Research Station at
Dollis Hill, building computers from scratch, and took evening classes
to get a degree in Mathematics.

In 1962 she set up a software company, employing almost entirely women,
working at home; the company was floated in 1996. Her team's projects
included programming Concorde's black box flight recorder. She adopted
the name "Steve" to fit in in a male domainated world.

http://www.bbc.co.uk/programmes/b05pmvl8
https://en.wikipedia.org/wiki/Steve_Shirley

Signed-off-by: Justin Cormack justin.cormack@unikernel.com

### CODE REVIEW

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -423,6 +423,9 @@ var (
 		// Carol Shaw - Originally an Atari employee, Carol Shaw is said to be the first female video game designer. https://en.wikipedia.org/wiki/Carol_Shaw_(video_game_designer)
 		"shaw",
 
+		// Dame Stephanie "Steve" Shirley - Founded a software company in 1962 employing women working from home. https://en.wikipedia.org/wiki/Steve_Shirley
+		"shirley"
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: you missed a comma here

**Comment by thaJeztah** [comment]
Impressive person 🙌

LGTM once the code is fixed

(note that, in general, each change to the list should have both a male, and female person, to keep the list in balance. however, we recently had a change that only added a male person, so this should be fine)

**Comment by justincormack** [comment]
Sorry typo fixed.

**Comment by vdemeester** [comment]
LGTM 👩

**Pull request merged**